src/v5424.rs

```diff
@@ -201,6 +226,33 @@ impl Formatter {
     }
 }
 
+#[cfg(feature = "chrono")]
+fn format_chrono_datetime<W: io::Write>(w: &mut W, datetime: ChronoLocalTime) -> io::Result<()> {
+    use chrono::Timelike;
+
+    const NANOSEC_IN_MILLI: u32 = 1000;
```

Comment:
There are 10e6 nano (1e-9) secs in a millisecond (1e-3). https://en.wikipedia.org/wiki/Orders_of_magnitude_(time).

Reply:
Resolved in https://github.com/tandemdrive/syslog-fmt/pull/18/commits/406df9d6047c14f3dd722cebb880613dd34edcce

Reply:
I think it should be something like `MICROS_IN_NANO_SEC`.

Reply:
Resolved in https://github.com/tandemdrive/syslog-fmt/pull/18/commits/290f6cbbf23676dfc1c5cc42815bc32f2136f3ea

Reply:
:+1: